adhocracy4/polls/assets/PollDashboard/EditPollManagement.jsx

```diff
@@ -0,0 +1,260 @@
+import React, { useState, useEffect } from 'react'
+import django from 'django'
+import dashboard from '../../../../adhocracy4/dashboard/assets/dashboard'
+import update from 'immutability-helper'
+import { EditPollQuestion } from './EditPollQuestion'
+import { EditPollOpenQuestion } from './EditPollOpenQuestion'
+import Alert from '../../../static/Alert'
+import EditPollDropdown from './EditPollDropdown'
+
+const api = require('adhocracy4').api
+const FlipMove = require('react-flip-move').default
```

Comment:
for understanding: why do we use require and not import for the api and flip move?

Reply:
I also wondered this, changing it broke it and so I changed it back intending to investigate after everything was working again but forgot, will take another look when I redo the other stuff

Reply:
I tested it yesterday and `import FlipMove from 'react-flip-move'` works but we need to add react-flip-move to the dev dependencies otherwise the linter will complain. For api `import api from '../.../../static/api' worked as well :)

Reply:
@goapunk thanks figured that, but do you know why it works with require but not import without it being in package.json? I've looked into it and I can't really figure it out?! would be good  need to include so much into a4 dev dependencies!

Reply:
Maybe require is not checked by the linter as it kind of executes the required file? And for package.json - if we ever have tests for this component we will have to add it anyway I think?

Reply:
just tested and `const test = require('thisDoesntExist')` doesn't trigger a linting error. I don't think that's very good as we won't catch mistakes as easy  like this